examples/quickstart/configs/training_azure.yaml

```diff
@@ -1,8 +1,12 @@
+enable_cache: True
+
 # Environment configuration
 settings:
   docker:
     parent_image: "zenmldocker/zenml-public-pipelines:quickstart-0.65.0-py3.11-azure"
     skip_build: True
+    # requirements: requirements.txt  # Uncomment this to add your own dependencies
+
 # Uncomment the following two lines to specify the accelerator for your skypilot vm orchestrator
 #  orchestrator.vm_azure:
 #    instance_type: 'Standard_NC6'
```

Comment:
FYI, skypilot is no longer the only orchestrator option for Azure, nor is it deployed by default anymore with the Azure one-click and terraform stack deployments. It was replaces by the AzureML orchestrator. If you need an equivalent configuration for AzureML, let me know.